### PR TITLE
forwarding_config updated

### PR DESCRIPTION
**Pull request by vktr-brlv**
forwarding_path argument added to forwarding_config block

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: variables.tf [hunk 1]
```diff
@@ -42,7 +42,7 @@ variable "project_id" {
 variable "target_name_server_addresses" {
   description = "List of target name servers for forwarding zone."
   default     = []
-  type        = list(string)
+  type        = list(object({ ipv4_address = string, forwarding_path = string }))
```
Comment (morgante) [inline variables.tf:45]: Please make this `list(map(any))`.

**Review by morgante — CHANGES_REQUESTED** [review]

File: variables.tf (hunk 1 above)
Comment (vktr-brlv) [reply variables.tf]: done

**Review by morgante — APPROVED** [review]

File: variables.tf (hunk 1 above)
Comment (byronmccollum) [reply variables.tf]: Why not `list(object({ ... }))`?

**Pull request merged**